python-package/lightgbm/basic.py

```diff
@@ -21,13 +21,21 @@
 from .libpath import find_lib_path
 
 
+def _log_callback(msg):
+    """Redirect logs from native library into Python console"""
```

Comment:
To make pytest docstring checks happy. It doesn't raise error right now because by default it doesn't check private members.
```suggestion
    """Redirect logs from native library into Python console."""
```